stdlib/public/core/Sort.swift

```diff
@@ -516,35 +516,34 @@ internal func _siftDown<C: MutableCollection & RandomAccessCollection>(
   subRange range: Range<C.Index>,
   by areInIncreasingOrder: (C.Element, C.Element) throws -> Bool
 ) rethrows {
-
-  let countToIndex = elements.distance(from: range.lowerBound, to: index)
-  let countFromIndex = elements.distance(from: index, to: range.upperBound)
-  // Check if left child is within bounds. If not, return, because there are
+  var index = index
```

Comment:
I'd suggest using `i` instead of `index`. Really, all these free functions ought to be converted to be extensions on `MutableCollection where Self: RandomAccessCollection`, and if that refactoring happens then `index` with clash with all the same-named methods on `self`.